src/include/homestore/btree/detail/varlen_node.hpp

```diff
@@ -166,7 +166,7 @@ class VariableNode : public BtreeNode {
             get_nth_value(ind_s - 1, &last_1_val, false);
             this->set_edge_value(last_1_val);
 
-            for (uint32_t i = ind_s; i < total_entries; i++) {
+            for (uint32_t i = ind_s - 1; i < total_entries; i++) {
```

Comment:
wrong calculation of available size.
When removing the edge entity happens, the `entity[ind_s-1]` is set as a new edge and removed as the normal entity. So its size must be released because it is set as a new edge.